client/containers/DashboardOverview/CollectionOverview/CollectionControls.tsx

```diff
@@ -30,22 +30,19 @@ const CollectionControls = (props: Props) => {
 			<ButtonGroup>
 				<MenuButton
 					aria-label="Set collection public or private"
-					// @ts-expect-error ts-migrate(2322) FIXME: Type 'string' is not assignable to type 'null | un... Remove this comment to see the full error message
 					buttonContent={isPublic ? 'Public' : 'Private'}
 					buttonProps={{
+						// @ts-expect-error: "lock2" not valid in TS definitions
```

Comment:
We will eventually have to change this so it accepts all the icon values provided in our own custom `components/Icon`.